tests/playwright-test/runner.spec.ts

```diff
@@ -449,3 +449,18 @@ test('should not crash with duplicate titles and line filter', async ({ runInlin
     `   - example.spec.ts:8`,
   ].join('\n'));
 });
+
+test('should allow wildcards in project option', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
+    'playwright.config.ts': `
+    module.exports = { name: 'project-name' };
+    `,
+    'a.test.js': `
+      const { test } = pwt;
+      test('one', async ({}) => {
+        expect(1).toBe(1);
+      });    `
+  }, { project: '.*oj.*t-na.?e' }, {}, {});
```

Comment:
Nit: '.*oj.*t-Na.?e' or `name: 'Project-Name'` to make sure the match is case-insensitive.